docs/Customizing Look and Functionality/Customizing Help Prompts/index.md

```diff
@@ -75,6 +75,21 @@ sdlManager.sendRPC(setGlobalProperties);
 ```
 !@
 
+@![javascript]
+```js
+const setGlobalProperties = new SDL.rpc.messages.SetGlobalProperties();
+setGlobalProperties.setVrHelpTitle('What Can I Say?');
+
+const item1 = new SDL.rpc.structs.VrHelpItem().setText("Show Artists").setPosition(1).setImage(<#image#>); // a previously uploaded image or null
+
+const item2 = new SDL.rpc.structs.VrHelpItem().setText("Show Albums").setPosition(2).setImage(<#image#>); // a previously uploaded image or null
+
+setGlobalProperties.setVrHelp([item1, item2]);
+// The help menu is updated
```

Comment:
This comment doesn't seem to be correct, shouldn't it be after the RPC is sent and a successful response is received?